invisible_flow/entities/data_officer.py

```diff
@@ -0,0 +1,13 @@
+from attr import dataclass
+
+
+@dataclass
+class Officer:
+    birth_year: str
+    first_name: str
+    gender: str
+    last_name: str
+    middle_initial: str
+    race: str
+    rank: str
+    cr_id: str
```

Comment:
What's `cr_id` ?

Reply:
cr_id is the column that the LOG_NO column is converted to that allows us the ability to make queries across tables with the same original LOG_NO value